R/rebuild.R

```diff
@@ -34,7 +34,9 @@ reset_search_path <- function() {
 }
 
 attach <- function(locked_package) {
-  if (!identical(locked_package$load, FALSE) && !locked_package$is_dependency_package) {
+  if (is.autoinstall_package(locked_package)) {
```

Comment:
In that case, this should look like:

```r
if (!identical(locked_package$load, FALSE)) {
  if (is.autoinstall_package(locked_package)) {
    load_package(locked_package)
  } else if (!locked_package$is_dependency_package) {
    library(locked_package$name, character.only = TRUE)
  }
}
```

You could even make a `load_package.locked_package` S3 specialization be the `library()` call.